redux/store.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
```

Comment:
I've seen some different examples of where to put the store file, @mnicole, would we want to just put it in a `store/` directory? [This](https://github.com/mcnamee/react-native-starter-kit) template seems to be fairly popular and they put the store config in `store/index.js`. Thoughts?

Reply:
yeah, that's where we put ours!

Reply:
Nice! 

So @ernewtoner, rename the file to `index.js` and move to a `store` dir. I'd recommend doing most of this with git since you're tracking the file and `redux` directory already:
```
git mv redux/store.js index.js
# remove the redux dir and create new dir called store
git mv redux/index.js store/
```

Reply:
Do you know why people do this? It doesn't seem like we'll have anything else in the store directory, store.js seems more descriptive than store/index.js.

Reply:
If for whatever reason we need to set up more stores, this is a good structure to have. As projects get more complex, it's helpful to have a project structure beyond individual files named whatever. That's my 2c anyway.